musl does not allow using <netinet/*> together with <linux/*> headers.
Since there are both netinet/if_ether.h and linux/if_ether.h providing
the same definitions, use linux/if_ether.h whenever other linux/*
headers are included.

Signed-off-by: Alex Suykov <alex.suykov@gmail.com>

--- aiccu/common/common.h
+++ aiccu/common/common.h
@@ -91,11 +91,12 @@
 	#include <sys/select.h>
 
 	#include <net/if.h>
-	#include <netinet/if_ether.h>
 #ifdef linux
 	#include <netpacket/packet.h>
+	#include <linux/if_ether.h>
 	#include <linux/if_tun.h>
 #else
+	#include <netinet/if_ether.h>
 #ifdef _DFBSD
 	#include <net/tun/if_tun.h>
 #else
